cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java

```diff
@@ -884,6 +886,42 @@ private static void setupHistoryDaysToLive() {
 		}
 	}
 
+
+	private static void setupMaxLimitForNumberOfProcessesPerWorker() {
+		worker_max_num_running_procs = getPreset("worker_max_num_running_procs");
+
+		if (worker_max_num_running_procs == null) {
+			worker_max_num_running_procs = getPreset("default_worker_max_num_running_procs");
+		}
+
+		// make sure preset is valid positive integer
+		try {
+			if (Integer.parseInt(worker_max_num_running_procs) <= 0) {
+				log.warn("Processes per worker value must be a positive integer. Got: " + worker_max_num_running_procs + ". Defaulting to 25.");
+				worker_max_num_running_procs = "25";
```

Comment:
I'm not sure I'm okay with setting a default of 25 like this.  I think the default should actually be based on something  tangible like number of CPU cores, like is done elsewhere in the code.

Reply:
```
worker_max_num_running_procs = CORES + "";
```

https://github.com/NASA-AMMOS/common-workflow-service/pull/126/commits/8ca402b4b5da5baf61729b41b4f2881ace82ff8a